src/main/java/com/kainos/ea/models/Role.java

```diff
@@ -0,0 +1,103 @@
+package com.kainos.ea.models;
+
+import com.kainos.ea.enums.Bands;
+import com.kainos.ea.enums.Capabilities;
+import com.kainos.ea.enums.Locations;
+import com.kainos.ea.enums.RoleStatus;
+
+import java.sql.Timestamp;
+import java.text.SimpleDateFormat;
+
+public class Role {
+    int jobRoleId;
```

Comment:
These fields should probably all be private and then have access via getters/setters?

Reply:
done